test/arrays/literals/hetLiteralsFirstGoverns.chpl

```diff
@@ -0,0 +1,5 @@
+var A = [1, 2, true, 4, false];
+var B = [1.0, 2, 3.0, 4, 5.0];
+
+writeln(A, ": ", A.type:string);
+writeln(B, ": ", B.type:string);
```

Comment:
The name of this test suggests that "first governs" is a thing.

I suggest moving the contents of this test into `hetLiteralsUnifiable.chpl` and removing this test.